rust/src/ike/ikev1.rs

```diff
@@ -0,0 +1,157 @@
+/* Copyright (C) 2020 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+// Author: Frank Honza <frank.honza@dcso.de>
+
+use std;
+use crate::core::{STREAM_TOSERVER};
+use crate::applayer::*;
+use nom;
+use crate::ike::parser::*;
+use crate::ike::ike::{IKEState, IkeEvent};
+use std::collections::HashSet;
+use crate::common::to_hex;
+
```

Comment:
Could you run `rustfmt rust/src/ike/*.rs` on some of these files ?

Reply:
@jasonish  do you know if we could add this `rustfmt` as a CI check like clang-format ?

Reply:
> @jasonish do you know if we could add this `rustfmt` as a CI check like clang-format ?

No, not really.  Rustfmt can only operate on complete files, not just changes like clang.  So we can't do much until we reformat all the code.

Reply:
Thanks Jason.
Franck, can you run rustfmt on some of these whole files ?

Reply:
yes, i'm just waiting a bit for further comments and would incorporate everything in v12?